components/alibi-detect-server/adserver/cd_model.py

```diff
@@ -37,18 +37,40 @@ def _drift_to_metrics(drift):
             {"key": "seldon_metric_drift_is_drift", "value": is_drift, "type": "GAUGE",}
```

Comment:
*is_drift* type is either **int** or **np.ndarray** (in the case of feature-wise drift for the KS detector). Needs to support both (optionally iterate through array).